uclapi.openapi.json

```diff
@@ -0,0 +1,4256 @@
+          },
+          "roomid" : {
+            "type" : "string",
+            "example" : "Z4",
+            "description": "The room ID (not to be confused with the roomname)."
+          },
+          "siteid" : {
+            "type" : "string",
+            "example" : "005",
+            "description": "Every room is inside a site (building). All sites have IDs."
+          },
+          "sitename" : {
+            "type" : "string",
+            "example" : "Main Building",
+            "description": "Every site (building) has a name. In some cases this is contained in the roomname as well."
+          },
+          "capacity" : {
+            "type" : "string",
+            "example" : 50,
+            "description": "Every room has a set capacity of how many people can fit inside it. When supplied, all rooms with the given capacity or greater will be returned."
+          },
+          "classification" : {
+            "type" : "string",
+            "example" : "SS",
+            "description": "The room type ID."
+          },
+          "classification_name" : {
+            "type" : "string",
+            "example" : "Social Space",
+            "description": "A human-readable version of the room type. AN = Anechoic Chamber, CI = Clinic Room, CF = Catering Facilities CFE = Cafe, CL = Cloakroom, CR = Classroom, ER = Equipment Room, IN = Installation, LA = Laboratory, LB = Library, LT = Lecture Theatre, MR = Meeting Room, OF = Office, PC1 = Public Cluster, PC2 = Public Cluster - Tutorial, PC3 = Public Cluster - Students, RC = Reverberation Chamber, SS = Social Space, STU = Studio, TH = Theatre. If the room type is unknown, this value will be set to 'Unknown Room Type'."
+          },
+          "automated" : {
+            "type" : "string",
+            "example" : "N",
+            "description": "Whether bookings in this room will be confirmed automatically. A stands for automated, and N for not automated. P represents that the confirmation will be automatic, but only under certain circumstances."
+          },
+          "location" : {
+            "type" : "object",
+            "properties" : {
+              "address" : {
+                "type" : "array",
+                "minItems" : 4,
+                "maxItems" : 4,
+                "items" : {
+                  "type" : "string"
+                },
+                "example" : [ "Gower Street", "London", "WC1E 6BT" ],
+                "description": "Address represented as an array."
+              },
+              "coordinates" : {
+                "type" : "object",
+                "properties" : {
+                  "lat" : {
+                    "type" : "number",
+                    "example" : "51.524699",
+                    "description": "Latitude of location"
+                  },
+                  "lng" : {
+                    "type" : "number",
+                    "example" : "-0.13366",
```

Comment:
nit: can these numbers be outside of quotes like above (or do floats cause problems)?